readthedocs/api/v2/templates/restapi/footer.html

```diff
@@ -41,8 +41,8 @@
       <dl>
         <dt>{% trans "Versions" %}</dt>
         {% for version in versions %}
-        <dd {% if version.verbose_name == current_version %} class="rtd-current-item" {% endif %}>
-          <a href="{{ version.get_subdomain_url }}{{ path|default_if_none:"" }}">{{ version.slug }}</a>
+        <dd {% if version == current_version %} class="rtd-current-item" {% endif %}>
+          <a href="{{ version.get_subdomain_url }}{{ path|default_if_none:"" }}">{{ version.explicit_name }}</a>
```

Comment:
I don't think this is needed -- we will never show a PR build in this list. The slug is what will be shown in the URL, so I think we want the slug?

Reply:
We were showhing the name for the current version, and the slug for the other versions, I think it makes sense showing the verbose name as this is how versions are originally named.